samples/devworkspace-controller.yaml

```
@@ -0,0 +1,206 @@
+          image: 'quay.io/samsahai/curl:latest'
+          volumeMounts:
+            - path: "/plugins"
+              name: plugins
+
+      - name: remote-endpoint
+        volume: {}
+          # ephemeral: true                #### We should add it in the Devfile 2.0 spec ! Not critical to implement at start though
+
+      - name: remote-runtime-injector
+        attributes:
+          "app.kubernetes.io/part-of": che-theia.eclipse.org
+          "app.kubernetes.io/component": bootstrapper
+        container:                          #### corresponds to `initContainer` definition in old meta.yaml.
+          image: "quay.io/eclipse/che-theia-endpoint-runtime-binary:7.20.0"
+          volumeMounts:
+            - path: "/remote-endpoint"
+              name: remote-endpoint
+          env:
+            - name: PLUGIN_REMOTE_ENDPOINT_EXECUTABLE
+              value: /remote-endpoint/plugin-remote-endpoint
+            - name: REMOTE_ENDPOINT_VOLUME_NAME
+              value: remote-endpoint
+            - name: PROJECTS_ROOT # Automatically adding this built-in is not implemented yet
+              value: "/projects"
+
+      - name: che-machine-exec
+        attributes:
+          "app.kubernetes.io/name": che-terminal.eclipse.org
+          "app.kubernetes.io/part-of": che.eclipse.org
+          "app.kubernetes.io/component": terminal
+        container:
+          image: "quay.io/eclipse/che-machine-exec:7.20.0"
+          command: ['/go/bin/che-machine-exec']
+          args:
+            - '--url'
+            - '0.0.0.0:4444'
+            - '--pod-selector'
+            - controller.devfile.io/workspace_id=$(CHE_WORKSPACE_ID)
+          endpoints:
+            - name: "che-mach-exec"
+              exposure: public
+              targetPort: 4444
+              protocol: ws
+              secure: true
+              attributes:
+                type: terminal
+      - name: vscode-go
+        attributes:
+          "app.kubernetes.io/part-of": che-theia.eclipse.org
+          "app.kubernetes.io/component": vscode-plugin
+          
+          # Added by Che-theia at start when detecting, after cloning, that the extensions.json in the repo
+          # contains the typescript vscode plugin.
+          "che-theia.eclipse.org/vscode-extensions":
+            - https://github.com/golang/vscode-go/releases/download/v0.16.1/go-0.16.1.vsix
+
+        container:
+          image: "quay.io/eclipse/che-sidecar-go:1.14.4-e8a574b"
+          memoryLimit: '512Mi'
+          mountSources: true
+          env:
+            - name: GOPATH
+              value: /go
+            - name: PLUGIN_REMOTE_ENDPOINT_EXECUTABLE
+              value: /remote-endpoint/plugin-remote-endpoint 
+            - name: THEIA_PLUGINS
+              value: local-dir:///plugins/sidecars/vscode-typescript
```

```suggestion
              value: local-dir:///plugins/sidecars/vscode-go
```
This is the part I was talking about the standup, I did the same thing in vscode-go